src/applications/claims-status/containers/ClaimStatusPage.jsx

```diff
@@ -85,20 +84,19 @@ const generatePhases = claim => {
       type: 'phase_entered',
       // We are assuming here that each phaseKey is of the format:
       // phaseXCompleteDate, where X is some integer between 1 and 7
+      // NOTE: Adding 1 because the a phases completion date is
+      // analagous to the phase after it's start date
+      // eg. phase1CompleteDate = start date for phase 2
       phase: Number(phaseKey.match(regex)[0]) + 1,
       date: previousPhases[phaseKey],
     });
   });
 
-  if (claim.attributes.closeDate !== null) {
```

Comment:
If the `closeDate !== null`, the timeline is not currently shown anyways, there is no point in including this code